autoload/maktaba/plugin.vim

```diff
@@ -102,6 +111,30 @@ function! s:SanitizedName(name) abort
 endfunction
 
 
+""
+" Gets a dictionary of {leaf path} for each path found on &rtp that does not
+" correspond to a registered plugin.
+function! s:GetUnregisteredLeafdirs() abort
+  let l:cache = s:unregistered_leafdirs_cache
+  let l:plugin_locations = keys(s:plugins_by_location)
+  if !has_key(l:cache, 'rtp') || !has_key(l:cache, 'plugin_locations') ||
+      \ l:cache.rtp isnot# &rtp ||
```

Comment:
`&rtp` is a string, so `!=#` would work equally well here, no?

(I'm never entirely sure whether `is#` was intended to be always equivalent to `==#` for strings.)